taipy/gui/gui.py

```diff
@@ -411,6 +420,33 @@ def __init__(
             for library in libraries:
                 Gui.add_library(library)
 
+    def _load_scripts(self, script_paths: t.Optional[t.Union[str, t.List[t.Union[str, Path]]]]):
+        if script_paths is None:
+            return
+
+        if isinstance(script_paths, (str, Path)):
+            script_paths = [script_paths]
+
+        for script_path in script_paths:
+            self._load_script(script_path)
+
+        if not self.__script_files:
+            raise Exception("No JavaScript files found in the specified script paths.")
+
+    def _load_script(self, script_path: t.Union[str, Path]):
+        if isinstance(script_path, str):
+            parsed_url = urlparse(script_path)
+            if parsed_url.netloc:
+                self.__script_files.append(script_path)
+                return
+            script_path = Path(script_path)
+
+        if script_path.exists() and script_path.is_file() and script_path.suffix == ".js":
+            self.__script_files.append(str(script_path))
+        else:
+            raise ValueError(f"Script path '{script_path}' does not exist, is not a file, or is not a JavaScript file.")
```

Comment:
Do we really want to break the app ? Or would a `_warn`be enough ?

Reply:
👍👍👍